frontend/views/containers/chatroom/ChatMain.vue

```diff
@@ -415,8 +418,8 @@ export default ({
         onlyRenderMessage: true // NOTE: DO NOT RENAME THIS OR CHATROOM WOULD BREAK
       }
     },
-    async renderMoreMessages (refresh = false) {
-      if (refresh) {
+    async renderMoreMessages (shouldInitiate = true) {
```

Comment:
Can you add a comment above this line explaining what `shouldInitiate` means?

Also maybe next to `messagesInitiated` on line 165?

Reply:
Sure. Will add comment.